### PR TITLE
feat implemented

### PR DESCRIPTION
**Pull request by lukasbuenger**
Well, not that much, I guess

### CODE REVIEW

File: components/editor/modules/link/ui.js [hunk 1]
```diff
@@ -120,6 +121,24 @@ export default ({TYPE}) => {
...
+    const repoChange = (onChange, value, node) => repo => {
+      onChange(
+        value.change().replaceNodeByKey(
+          node.key,
+          {
+            type: TYPE,
+            kind: 'inline',
+            data: node.data.merge({
+              title: repo.text,
+              href: `/~${repo.value.id}?autoSlug`
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js]: surplus tide—add github.com link here

File: components/editor/modules/link/ui.js [hunk 2]
```diff
@@ -120,6 +121,24 @@ export default ({TYPE}) => {
...
     }
+    const repoChange = (onChange, value, node) => repo => {
+      onChange(
+        value.change().replaceNodeByKey(
+          node.key,
+          {
+            type: TYPE,
+            kind: 'inline',
+            data: node.data.merge({
+              title: repo.text,
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js]: looks wrong, probably repo.value.meta.title or something

File: components/editor/modules/link/ui.js [hunk 3]
```diff
@@ -120,6 +121,24 @@ export default ({TYPE}) => {
         )
       )
     }
+    const repoChange = (onChange, value, node) => repo => {
+      onChange(
+        value.change().replaceNodeByKey(
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js]: imo just update the data and do not replace text/node for article links

File: components/editor/modules/link/ui.js [hunk 4]
```diff
@@ -140,6 +159,10 @@ export default ({TYPE}) => {
                   onChange={onInputChange('title')}
                 />
                 <SearchUserForm onChange={authorChange(onChange, value, node)} />
+                <RepoSearch
+                  label={t('link/repo/search', undefined, 'Artikel suchen')}
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js]: why the default? Please add it to the sheet https://docs.google.com/spreadsheets/d/1qPubCKz2YzbcFkRfF0MrgRqD0KLsgRPxoZfG5G3tHOA/edit#gid=0 and run `npm run translations`

File: components/editor/modules/link/ui.js (hunk 2 above)
Comment (lukasbuenger) [reply components/editor/modules/link/ui.js]: Nope. You get a RepoSearch result here and the text always equals the repo title.

**Comment by lukasbuenger** [comment]
I know about translations, just wait for it.

**Comment by lukasbuenger** [comment]
Ok, should be there. Merge?

**Comment by tpreusse** [comment]
thank you for adding this

**Pull request merged**